### PR TITLE
Warning messages and error handling

### PR DESCRIPTION
**Pull request by manodnyab**
## Description

1) Added a warning if the task role with the required permissions is not set.
2) String changes in the menu options
3) Adding help links 

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecUtils.kt [hunk 1]
```diff
@@ -24,12 +26,20 @@ import software.aws.toolkits.telemetry.Result
...
-                .enableExecuteCommand(enabled)
-                .forceNewDeployment(true).build()
-            project.awsClient<EcsClient>().updateService(request)
+            try {
+                val request = UpdateServiceRequest.builder()
+                    .cluster(service.clusterArn())
+                    .service(service.serviceName())
+                    .enableExecuteCommand(enabled)
+                    .forceNewDeployment(true).build()
+                project.awsClient<EcsClient>().updateService(request)
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecUtils.kt:35]: FYI: You can inline the request building by passing a lambda to updateService

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecUtils.kt [hunk 2]
```diff
@@ -24,12 +26,20 @@ import software.aws.toolkits.telemetry.Result
...
+                checkServiceState(project, service, enable = enabled)
+            } catch (e: InvalidParameterException) {
+                runInEdt {
+                    TaskRoleNotFoundWarningDialog(project).show()
+                    EcsTelemetry.enableExecuteCommand(project, Result.Failed)
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecUtils.kt:40]: where do we have the Success published?

Comment (manodnyab) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecUtils.kt:40]: After the call has been made and the service has reached final state

**Review by abrooksv — APPROVED** [review]

**Pull request merged**